<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="RectView" name="rectview"/>
    </item>
    <item>
     <widget class="RTreeView" name="rtreeview"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddRectangle"/>
   <addaction name="actionRemoveRectangle"/>
   <addaction name="actionSearchRect"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAddRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Rectangle</string>
   </property>
  </action>
  <action name="actionRemoveRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove Rectangle</string>
   </property>
  </action>
  <action name="actionSearchRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search Rectangle</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RectView</class>
   <extends>QGraphicsView</extends>
   <header>views.h</header>
  </customwidget>
  <customwidget>
   <class>RTreeView</class>
   <extends>QGraphicsView</extends>
   <header>views.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
